src/extension.ts

```diff
@@ -6,13 +6,16 @@ import { makeClient } from './client'
 import { Config, LanguageConfig, alloglot } from './config'
 import { makeFormatter } from './formatter'
 import { makeTags } from './tags'
+import { readFileSync, writeFile, writeFileSync } from 'fs'
 
 export function activate(context: vscode.ExtensionContext): void {
   const settingsSection = vscode.workspace.getConfiguration(alloglot.root)
+  const alloglotWorkspaceLanguages = getWorkspaceConfig('.vscode/alloglot.json');
+  const alloglotVscodeLanguages = settingsSection.get<Array<LanguageConfig>>(alloglot.config.languages, [])
+  const alloglotLanguages = alloglotVscodeLanguages.length === 0 ? alloglotWorkspaceLanguages : alloglotVscodeLanguages;
```

Comment:
ah, so it's either wholesale override, rather than merge. that's a lot easier to implement, and more predictable for end users.